pom.xml

```diff
@@ -147,7 +150,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-shade-plugin</artifactId>
-        <version>3.5.1</version>
+        <version>3.6.0</version>
```

Comment:
Already defined in our parent pom: https://github.com/SonarSource/parent-oss/blob/d402f7ac11f58e0d5b3e62be033f673f3b0fa2e3/pom.xml#L104